### PR TITLE
Plazma

### PR DESCRIPTION
**Pull request by DaroDalya**

### CODE REVIEW

**Comment by zigen** [comment]
:lgtm:

**Review by zigen — APPROVED** [review]

File: input-counter/main.js [hunk 1]
```diff
@@ -1 +1,29 @@
-// write conde in here
\ No newline at end of file
+// write conde in here
+const plusButton = document.getElementById('plus')
+const minusButton = document.getElementById('minus')
+const resetButton = document.getElementById('reset')
+const input = document.getElementById('numberInput')
+const numberElement = document.getElementById('number')
+
+
```
Comment (hrl7) [inline input-counter/main.js]: do you need empty 2 lines ?

File: input-counter/main.js [hunk 2]
```diff
@@ -1 +1,29 @@
...
+function changeValue(text){
```
Comment (hrl7) [inline input-counter/main.js]: does this function change value? what value changed?

**Review by hrl7 — CHANGES_REQUESTED** [review]
explain my questions or fix your code

**Review by hrl7 — APPROVED** [review]

**Pull request closed without merging**